pages/index.js

```diff
@@ -21,6 +22,19 @@ export default function Home() {
         </p>
       </main>
 
+      <div className="pt-4">
+        <ActionButton
+          // id={id + "-MoreInfo"}
+          text="Action button"
+          notFullRound={false}
+          //  rounded={true}
+          invert={false}
+          className={"font-semibold"}
+          disabled={false}
+          dataCyButton="action-button"
+        />
+      </div>
```

Comment:
I think this might have been left in by accident

Reply:
I actually left it there on purpose so you could see how to use it and try it out. Eventually it will not be there at this location i imagine. If you prefer to remove it now, feel free to do so.